### DISCUSSION TITLE
Server Config Requirements

### DISCUSSION THREAD

**Discussion by Maelep** [wolveix/satisfactory-server#29] [Q&A] [answered]
Do you know the required configuration in terms of CPU and RAM?

**Comment by wolveix** [comment, marked as answer]
Any modern CPU should be able to run this without an issue. The RAM requirement depends on the amount of concurrent players, and where you are in the game. You typically need anywhere from 1GB - 4GB, sometimes less. You also need 20GB of storage available for the game files.

**Reply by Maelep** [reply]
Thank you for your answer.
